### ISSUE TITLE
mlflow checkpoints in the wrong location 

### ISSUE THREAD

**Issue by david-waterworth** [PyTorchLightning/pytorch-lightning#2939]
I'm not sure if I'm doing something wrong, I'm using mlflow instead of tensorboard as a logger. I've used the defaults i.e.

```
mlflow = loggers.MLFlowLogger()
trainer = pl.Trainer.from_argparse_args(args, logger=mlflow)
```

I'm ending up with the following folder structure

\mlflow
\mlflow\1
\mlflow\1\\{guid}\artifacts
\mlflow\1\\{guid}\metrics
\mlflow\1\\{guid}\params
\mlflow\1\\{guid}\meta.yaml
**\1\\{guid}\checkpoints**

i.e. the checkpoints are in the wrong location, they should be in the `\mlflow` folder. 

Perhaps this is an mlflow rather than pytorch-lightning issue? 

I'm using pytorch-lightning 0.8.5 on macos running in python 3.7.6

**Comment by Borda** [comment]
@david-waterworth mind try the latest 0.9rc12?

**Comment by awaelchli** [comment]
It was fixed here: #2502 
The checkpoints subfolder will go here: `mlflow\1{guid}\checkpoints`, is that what you want @david-waterworth ?

**Comment by david-waterworth** [comment]
Thanks @awaelchli  yes that's what I want - thanks!
